quipucords/fingerprinter/__init__.py

```diff
@@ -649,6 +649,18 @@ def _process_network_fact(source, fact):
                                 fingerprint,
                                 fact_value=system_creation_date.date())
 
+    if fact.get('connection_timestamp'):
```

Comment:
This feels like it could be put into a helper function.  Perhaps there would be too many parameters.